apps/sap-commerce-cloud/README.md

```diff
@@ -1,3 +1,31 @@
 # SAP Commerce Cloud App
 
-TODO: fill this out
+An app that allows you to connect your SAP Commerce Cloud catalogue with Contentful.
+
+## Installation and Usage
+
+See the [Contentful Connector for SAP Commerce Cloud Marketplace listing](https://www.contentful.com/marketplace/sap-commerce-cloud-connector) for information about how to install and use the app. Also view the two help center documentation resources below for additional information:
+
+- [SAP App guide](https://www.contentful.com/help/sap-app)
+- [Configuring base site URL in backoffice](https://help.sap.com/docs/SAP_COMMERCE_CLOUD_PUBLIC_CLOUD/0fa6bcf4736c46f78c248512391eb467/07f7eaaf85b94ee9b2ded952880379ef.html)
+
+### Frontend
+
+The frontend lives in the `frontend/` folder.
+Instructions for installing or updating the app and running tests can be found in the README file for the frontend.
+
+### Available scripts
```

Comment:
Either in this section or its own, or move this to the app-actions readme..but we should have a section for "Testing hosted app actions" similar to this: https://github.com/contentful/apps/blob/master/apps/microsoft-teams/README.md#testing-backend-actions

Here's some content:
```
1. Build the app: Run `npm run build` from the root directory of the app (or just `npm run build-app-actions` or `npm run build-frontend` if you've only made changes in one of those directories and don't want to rebuild the whole app)
2. Deploy to staging: Run either `npm run deploy:test:sap` or `npm run deploy:test:sap-air` depending on which app you want to use for testing (Note: you will need to pass in DEV_TESTING_ORG_ID and  TEST_CMA_TOKEN as environment variables)
3. Call the app action: For example

$ npm run call-app-action -- --appActionId=fetchProductPreviews --params='{ <params list here> }' --spaceId=<space_id> --accessToken=<access_token>
```

Reply:
@mgoudy91 can you drop your params example here for the `fetchProductPreviews`?

Reply:
`npm run call-app-action -- --appActionId=fetchProductPreview --params='{ "skus": "[\"https://api.cm77gs48zv-contentfu1-d1-public.model-t.cc.commerce.ondemand.com/occ/v2/blah/products/CONF_SCREWDRIVER_S\", \"https://api.cm77gs48zv-contentfu1-d1-public.model-t.cc.commerce.ondemand.com/occ/v2/powertools-spa/products/MZ-FG-E101\"]" }' --spaceId=a5tfsecio4yc --accessToken=CFPAT-rbBpJ1_KNLU_viBJu88UigtLYSOw1_mvvBe09ZSHIpc
`

Reply:
`npm run call-app-action -- --appActionId=fetchProductPreview --params='{ "skus": "[\"https://api.cm77gs48zv-contentfu1-d1-public.model-t.cc.commerce.ondemand.com/occ/v2/blah/products/CONF_SCREWDRIVER_S\", \"https://api.cm77gs48zv-contentfu1-d1-public.model-t.cc.commerce.ondemand.com/occ/v2/powertools-spa/products/MZ-FG-E101\"]" }' --spaceId=SPACE_ID --accessToken=TOKEN_HERE
`